faker/providers/person/en_NZ/__init__.py

```diff
@@ -0,0 +1,627 @@
+        ('West', 0.09),
+        ('Mason', 0.12),
+        ('Finch', 0.03),
+        ('Norton', 0.03),
+        ('Burke', 0.05),
+        ('Holden', 0.04),
+        ('Lee', 0.2),
+        ('Smart', 0.04),
+        ('Bull', 0.04),
+        ('Bryant', 0.04),
+        ('Gray', 0.12),
+        ('Watts', 0.08),
+        ('Brady', 0.03),
+        ('Baker', 0.2),
+        ('Barton', 0.05),
+        ('Davis', 0.17),
+        ('Baxter', 0.05),
+        ('Taylor', 0.53),
+        ('Carr', 0.07),
+        ('Wong', 0.04),
+        ('Cameron', 0.03),
+        ('Gardiner', 0.03),
+        ('Hawkins', 0.07),
+        ('Shaw', 0.15),
+        ('Wallace', 0.05),
+        ('Young', 0.16),
+        ('Shah', 0.06),
+        ('Gregory', 0.07),
+        ('Ball', 0.08),
+        ('Norman', 0.04),
+        ('Lawrence', 0.09),
+        ('Bowen', 0.04),
+        ('Wheeler', 0.05),
+        ('Bartlett', 0.04),
+        ('Sutton', 0.06),
+        ('Lyons', 0.03),
+        ('Hutchinson', 0.05),
+        ('Poole', 0.05),
+        ('Cooke', 0.06),
+        ('Franklin', 0.03),
+        ('Howe', 0.04),
+        ('Walker', 0.27),
+        ('Johnston', 0.05),
+        ('Austin', 0.05),
+        ('Chadwick', 0.03),
+        ('Bell', 0.15),
+        ('Wall', 0.04),
+        ('Woodward', 0.05),
+        ('Preston', 0.04),
+        ('Bennett', 0.16),
+        ('Murray', 0.1),
+        ('Begum', 0.13),
+        ('McDonald', 0.06),
+        ('Hudson', 0.07),
+        ('Cross', 0.06),
+        ('Singh', 0.13),
+        ('Howarth', 0.03),
+        ('Hewitt', 0.05),
+        ('Curtis', 0.06),
+        ('Harding', 0.07),
+        ('May', 0.05),
+        ('Wells', 0.07),
+        ('Giles', 0.03),
+        ('Watson', 0.17),
+        ('Nolan', 0.03),
+        ('Andrews', 0.09),
+        ('Hayward', 0.04),
+        ('Schofield', 0.04),
+        ('Hunt', 0.12),
+        ('Robson', 0.06),
+        ('Arnold', 0.05),
+        ('Morgan', 0.19),
+        ('Coates', 0.03),
+        ('Page', 0.07),
+        ('Simpson', 0.13),
+        ('Stewart', 0.09),
+        ('Robinson', 0.29),
+        ('Fleming', 0.03),
+        ('Scott', 0.18),
+        ('Chambers', 0.06),
+        ('Turner', 0.23),
+        ('Watkins', 0.06),
+
+        # Additional common NZ surnames
+        ('Henare', 0.4),
+        ('Ngata', 0.3),
+        ('Turei', 0.2),
+        ('Parata', 0.2),
+        ('Lomu', 0.1),
+        ('Hopoate', 0.08),
+        ('Palepoi', 0.08),
+        ('Savea', 0.07),
+        ('Tuilevu', 0.06),
+        ('Van Wyk', 0.3),
+        ('Van der Merwe', 0.2),
+        ('Van der Linde', 0.1),
+        ('Steenkamp', 0.2),
+        ('Dijkstra', 0.18),
+        ('Oosterhof', 0.15),
+        ('Wang', 0.6),
+        ('Li', 0.5),
+        ('Chen', 0.5),
+        ('Liu', 0.4),
+        ('Zhang', 0.3),
+        ('Huang', 0.2),
+        ('Wu', 0.2),
+        ('Yang', 0.1),
+        ('Xu', 0.1),
+        ('Wong', 0.07),
+        ('Zhou', 0.05),
+    ))
+
+    # No name suffixes in New Zealand
+    suffixes_female = ('',)
```

Comment:
These can be removed

Reply:
Removing these results in "Jr." or "III" on the end of people's names, which only happens in New Zealand in re-runs of *Dallas*.

Reply:
I take it back, you're right. That's odd, it wasn't behaving like that before. I blame __pycache__ :-)